src/components/calcite-button/calcite-button.tsx

```diff
@@ -194,6 +197,7 @@ export class CalciteButton {
       "icon-start",
       "icon-end",
       "id",
+      "isSplitPrimary",
```

Comment:
This should match `splitChild` above yeah?

Reply:
Totally! Thanks for catching